diff --git a/dwl.c b/dwl.c
index bbf9b6a..4f71e95 100644
--- a/dwl.c
+++ b/dwl.c
@@ -527,10 +527,7 @@ applyrules(Client *c)
 		}
 	}
 
-<<<<<<< LEFT
 	c->isfloating |= client_is_float_type(c);
-||||||| BASE
-=======
 	wl_list_for_each(m, &mons, link) {
 		// tag with different monitor selected by rules
 		if (m->tagset[m->seltags] & newtags) {
@@ -538,7 +535,6 @@ applyrules(Client *c)
 			break;
 		}
 	}
->>>>>>> RIGHT
 	setmon(c, mon, newtags);
 	attachclients(mon);
 }
@@ -2124,20 +2120,8 @@ monocle(Monitor *m)
 	int n = 0;
 
 	wl_list_for_each(c, &clients, link) {
-<<<<<<< LEFT
 		if (!VISIBLEON(c, m) || c->isfloating || c->isfullscreen)
 			continue;
-||||||| BASE
-			if (c->mon != m)
-				continue;
-			occ |= c->tags;
-			if (c->isurgent)
-				urg |= c->tags;
-=======
-			occ |= c->tags;
-			if (c->isurgent)
-				urg |= c->tags;
->>>>>>> RIGHT
 		resize(c, m->w, 0);
 		n++;
 	}
